docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -8,13 +8,13 @@ Components of any type can specify the endpoints that the container exposes. The
 
 . Specify endpoints properties as shown in the following example:
 +
-.Specify endpoints properties
+.Specifying endpoints properties
```

Comment:
"Specifying endpoints properties" might be a malformed plural possessive. Maybe "Specifying endpoint properties" instead?